packages/graphql/tests/integration/math.int.test.ts

```diff
@@ -0,0 +1,816 @@
+                id,
+            }
+        );
+        expect(storedValue.records[0].get("viewers")).toEqual(int(110));
+        expect(storedValue.records[0].get("length")).toEqual(int(90));
+    });
+
+    test("Should be possible to update nested nodes", async () => {
+        const initialViewers = int(100);
+        const name = "Luigino";
+        const movie = generateUniqueType("Movie");
+        const actor = generateUniqueType("Actor");
+        const typeDefs = `
+        type ${movie.name} {
+            viewers: Int!
+            workers: [${actor.name}!]! @relationship(type: "WORKED_IN", direction: IN)
+        }
+        type ${actor.name} {
+            id: ID!
+            name: String!
+            worksInMovies: [${movie.name}!]! @relationship(type: "WORKED_IN", direction: OUT)
+        }
+        `;
+
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+
+        const id = generate({
+            charset: "alphabetic",
+        });
+
+        const query = `
+        mutation($id: ID, $value: Int) {
+            ${actor.operations.update}(where: { id: $id }, 
+                update: {
+                    worksInMovies: [
+                    {
+                      update: {
+                        node: {
+                          viewers_INCREMENT: $value
+                        }
+                      }
+                    }
+                  ]
+                }
+              ) {
+                ${actor.plural} {
+                    name
+                    worksInMovies {
+                      viewers
+                    }
+                }
+            }
+        }
+        `;
+
+        try {
+            // Create new movie
+            await session.run(
+                `
+                CREATE (a:${movie.name} {viewers: $initialViewers}), (b:${actor.name} {id: $id, name: $name}) WITH a,b CREATE (a)<-[worksInMovies: WORKED_IN]-(b) RETURN a, worksInMovies, b
+                `,
+                {
+                    id,
+                    initialViewers,
+                    name,
+                }
+            );
+            // Update movie
+            const gqlResult = await graphql({
+                schema: await neoSchema.getSchema(),
+                source: query,
+                variableValues: { id, value: 10 },
+                contextValue: { driver, driverConfig: { bookmarks: session.lastBookmark() } },
+            });
+
+            expect(gqlResult.errors).toBeUndefined();
+            const storedValue = await session.run(
+                `
+                MATCH (n:${actor.name} {id: $id})--(m:${movie.name}) RETURN n.name AS name, m.viewers AS viewers
+                `,
+                {
+                    id,
+                }
+            );
+            expect(storedValue.records[0].get("viewers")).toEqual(int(110));
+            expect(storedValue.records[0].get("name")).toBe(name);
+        } finally {
+            await session.close();
+        }
```

Comment:
Isn't this session already being handled by `afterEach`?

```suggestion
```

Reply:
Absolutely, 
I thought I removed them all, but I was wrong 👍